src/main/scala/firrtl/passes/memlib/ReplaceMemMacros.scala

```diff
@@ -136,7 +137,11 @@ class ReplaceMemMacros(writer: ConfWriter) extends Transform with DependencyAPIM
     *  The wrapper module has the same type as the memory it replaces
     *  The external module
     */
-  def createMemModule(m: DefAnnotatedMemory, wrapperName: String): Seq[DefModule] = {
+  def createMemModule(
+    m:           DefAnnotatedMemory,
+    wrapperName: String,
+    insertFunc:  DefAnnotatedMemory => Unit
+  ): Seq[DefModule] = {
```

Comment:
As noted in other comments, this isn't binary compatible. But when making such changes, I'd prefer if the new methods are private since in many of these cases, it doesn't seem like a useful public API (making it easier to make such changes in the future).